reference/nats-protocol/nats-protocol/README.md

```diff
@@ -119,20 +134,26 @@ The `CONNECT` message is the client version of the [`INFO`](./#info) message. On
+| `echo`          | If set to `false`, the server (version 1.2.0+) will not send originating messages from this connection to its own subscriptions. Clients should set this to `false` only for server supporting this feature, which is when `proto` in the `INFO` protocol is set to at least `1`. | bool   | false                        |
+| `sig`           | In case the server has responded with a `nonce` on `INFO`, then a NATS client must use this field to reply with the signed `nonce`.                                                                                                                                               | string | if `nonce` received          |
+| `jwt`           | The JWT that identifies a user permissions and account.                                                                                                                                                                                                                           | string | false                        |
+| `no_responders` | Enable [quick replies for cases where a request is sent to a topic with no responders](nats-concepts/core-nats/reqreply#no_responders).                                                                                                                                           | bool   | false                        |
+| `headers`       | Whether the client supports headers.                                                                                                                                                                                                                                              | bool   | false                        |
+| `nkey`          | ???.                                                                                                                                                                                                                                                                              | string | false                        |
+| `account`       | ???.                                                                                                                                                                                                                                                                              | bool   | false                        |
+| `account_new`   | ???.                                                                                                                                                                                                                                                                              | bool   | false                        |
```

Comment:
unsure what these fields are used for without diffing into the server code. If someone does know of the top of their head that would be appreciated.